src/IB/IBFEInstrumentPanel.cpp

```diff
@@ -330,9 +351,85 @@ IBFEInstrumentPanel::readInstrumentData(const int U_data_idx,
         }
     }
 
+    // Interpolate the pressure at the centroid of each meter mesh.
+    //
+    // TODO: Factor out common code for finding assignments of points to patches.
```

Comment:
it would be nice to re-use FEDataManager here but I don't think its flexible enough - see also https://github.com/drwells/fiddle/pull/84